resources/add_to_list.rb

```diff
@@ -35,7 +35,7 @@
       next unless line =~ regex
 
       log "Impacted line: #{line}" do
-        level :debug
+        level :info
       end
```

Comment:
At this point the line matches the regex but it may not be changed.  Log creates output and a changed resource even if nothing else changes.